src/halmos/sevm.py

```diff
@@ -2037,9 +2052,13 @@ def mk_storagedata(self) -> StorageData:
         return self.storage_model.mk_storagedata()
 
     def sload(self, ex: Exec, addr: Any, loc: Word) -> Word:
-        return self.storage_model.load(ex, addr, loc)
+        val = self.storage_model.load(ex, addr, loc)
+        ex.context.trace.append(StorageRead(addr, loc, val))
+        return val
 
     def sstore(self, ex: Exec, addr: Any, loc: Any, val: Any) -> None:
+        ex.context.trace.append(StorageWrite(addr, loc, val))
+
```

Comment:
do you want to add this event even in the case of raising WriteInStaticContext below?

Reply:
yes, I think it's going to be useful for debugging - so you know what it tried to write that caused the revert

Reply:
example
<img width="1034" alt="image" src="https://github.com/user-attachments/assets/0a800d0f-d128-41aa-b8d6-8f5939bfb5a4" />